docs/source/en/modular_transformers.md

```diff
@@ -7,7 +7,7 @@ aspect of the library. This component's downside is that it limits the inheritan
 files to others in the toolkit.
 
 As a result, model components tend to be repeated across many files. There are as many attention layers defined
-in `transformers` as there are models, and a significant number of those are identical to each other. 
+in `transformers` as there are models, and a significant number of those are identical to each other.
```

Comment:
(sorry, my IDE removes trailing white spaces 😅 the actual diff is the last line)